libs/futures/CMakeLists.txt

```diff
@@ -54,6 +56,8 @@ add_hpx_module(
   SOURCES ${futures_sources}
   HEADERS ${futures_headers}
   COMPAT_HEADERS ${futures_compat_headers}
+  EXCLUDE_FROM_GLOBAL_HEADER "hpx/futures/detail/future_data.hpp"
```

Comment:
Not necessary anymore since we exclude all detail headers from the global one in [HPX_AddModule](https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_AddModule.cmake#L134)

Reply:
Could you open a PR to remove that, please?